examples/jsm/controls/OrbitControls.js

```diff
@@ -92,6 +92,7 @@ class OrbitControls extends Controls {
 		// Set to false to disable rotating
 		this.enableRotate = true;
 		this.rotateSpeed = 1.0;
+		this.keyRotateSpeed = 10.0;
```

Comment:
Would you mind not changing the default value so we don't change the behavior?

Reply:
Happy to change this default to 1 as it's now configurable. But .. The default of `1` is fairly slow when moving with the keyboard. Having a slightly faster speed, would be similar to `keyPanSpeed` which is also higher when using the keyboard (7 vs 1).

The following recordings is me just holding shift+<arrow> down, so no repeated presses:

keyRotateSpeed = 1

https://github.com/user-attachments/assets/4e488e40-f61b-4491-b386-2b41acfea5e7

keyRotateSpeed = 10

https://github.com/user-attachments/assets/773b8a5a-bc38-4232-97e5-0233ecbbb9d8

Let me know what you think, happy to go either way on this

Reply:
No worries at all, I pushed a commit which sets it to 1